name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/dialogs/transactions/AccountTransactionDialog.java

```diff
@@ -339,6 +347,9 @@ public void widgetSelected(SelectionEvent e)
         WarningMessages warnings = new WarningMessages(this);
         warnings.add(() -> model().getDate().isAfter(LocalDate.now()) ? Messages.MsgDateIsInTheFuture : null);
         model.addPropertyChangeListener(Properties.date.name(), e -> warnings.check());
+        
+        warnings.add(() -> model().getExDate().isAfter(model().getDate()) ? "Ex-Dividenden-Tag liegt nach dem Ausschüttungs-Tag" : null); //$NON-NLS-1$
```

Comment:
The translation should be transferred to messages.properties

Reply:
I added the warning-message in project *name.abuchen.portfolio.ui*. 
Is it necessary adding this in the project *name.abuchen.portfolio* ? @Nirus2000 

I also added empty messages for other languages.
Any advice translating the message to other languages? @Nirus2000

Reply:
i think no... if all translated.